### DISCUSSION TITLE
I Forked Homebrew and Reenabled Sudo

### DISCUSSION THREAD

**Discussion by JavaPerson2** [Homebrew/discussions#3050] [Everyday usage]
I thought this would be helpful because Homebrew disabled this in the settings. It's obviously not for everyone but if you were interested I'll post the fork.

**Comment by gromgit** [comment]
I'm curious: What problem are you solving, that "run Homebrew with sudo" is the appropriate solution?

**Comment by JavaPerson2** [comment]
Apple designed the terminal in a specific way so this needs to be reenabled.

**Comment by carlocab** [comment]
Removing the check for `sudo` makes Homebrew **unsafe**. You **do not** want to give root privileges to random build scripts when running `brew install --build-from-source`, for example.

If you want to be able to run `brew` as the root user, then there are many more changes that need to be made to `brew` so that it handles root privileges correctly. Chances are that several thousand formulae will need to be rewritten as well. Unless you've done this work, please don't share your fork, because this can be harmful to its users.
